repository/l.json

```diff
@@ -769,17 +769,21 @@
 		},
 		{
 			"name": "LESS",
```

Comment:
Maybe we should re-name to Less for consistency. 

I wonder if we could maybe also consolidate [Less​Improved](https://github.com/braver/LessImproved) by removing it and adding it to "previous names" here?

Reply:
:( Making all the tags and releases again.

Reply:
LessImproved isn't registered at packagecontrol.io. AFAIK, it won't be renamed by PC if it was installed manually.

Can add it to `previous_names` in case someone added the repo to PC, though.